icons/arduino/arduino-plain.svg

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="128px" height="128px" viewBox="0 0 128 128" version="1.1">
+<g >
+<path style="fill-rule: evenodd; stroke: none; fill-opacity: 1" d="M 0.3 66.5 L 0.3 56.9 C 0.5 56.8 0.5 56.6 0.5 56.4 C 3.5 43.3 11.7 35.4 24.7 32.3 C 25.8 32 27 32.1 28.1 31.7 L 34.5 31.7 C 34.6 31.9 34.9 31.8 35.1 31.8 C 41.5 32.5 47.3 34.8 52.4 38.8 C 56.8 42.1 60.2 46.3 63.3 50.8 C 63.7 51.4 63.9 51.4 64.3 50.8 C 66.1 48.2 68 45.7 70.2 43.4 C 75.5 37.7 81.9 33.7 89.7 32.3 C 90.8 32 92.1 32.1 93.2 31.7 L 99.4 31.7 C 99.5 31.9 99.7 31.8 99.9 31.8 C 101.8 32 103.6 32.4 105.4 32.9 C 118.8 36.8 128.3 49.1 127.5 63 C 126.9 74.7 121 83.1 110.7 88.4 C 105.6 91.2 100 91.9 94.2 91.8 C 86.6 91.7 80 89.1 74.3 84.1 C 70.3 80.6 67.2 76.4 64.3 72 C 63.9 71.4 63.7 71.5 63.3 72.1 C 61.5 74.8 59.6 77.5 57.4 80.1 C 53.5 84.5 49 88.1 43.4 90 C 36.5 92.4 29.5 92.5 22.5 90.6 C 12.4 87.7 5.5 81.3 1.7 71.5 C 1.1 69.9 0.8 68.1 0.3 66.5 Z M 32.1 81.2 C 37.8 81.4 42.7 79.5 46.9 75.6 C 51.2 71.6 54.3 66.6 57.4 61.7 C 57.5 61.4 57.5 61.2 57.3 60.9 C 54.7 56.8 52 52.7 48.4 49.3 C 41.5 42.7 33.4 40.5 24.3 43.4 C 16.8 45.9 12 51.2 10.9 59.2 C 9.8 66.7 12.7 72.7 18.7 77.2 C 22.7 80.1 27.2 81.3 32.1 81.2 Z M 95.5 81.2 C 97.7 81.2 99.9 81.1 102 80.5 C 109.9 78.1 115.1 73.2 116.6 65 C 118.1 56.9 115 50.4 108.2 45.8 C 100.7 40.6 89.8 41.1 82.2 46.8 C 77.1 50.6 73.7 55.7 70.3 61 C 70.1 61.3 70.2 61.5 70.3 61.8 C 73 66.1 75.7 70.4 79.2 74.1 C 83.6 78.8 88.9 81.5 95.5 81.2 Z M 95.5 81.2"/>
+<path style="fill-rule: evenodd; stroke: none; fill-opacity: 1" d="M 32 58.5 C 35.2 58.5 38.5 58.6 41.7 58.5 C 42.5 58.5 42.6 58.7 42.6 59.5 C 42.5 60.4 42.5 61.3 42.6 62.2 C 42.7 63 42.5 63.2 41.6 63.2 C 37.3 63.2 33 63.2 28.8 63.2 C 26.6 63.2 24.4 63.1 22.2 63.2 C 21.5 63.2 21.3 63 21.3 62.2 C 21.4 61.3 21.4 60.4 21.3 59.4 C 21.3 58.7 21.5 58.5 22.2 58.5 C 25.4 58.6 28.7 58.5 32 58.5 Z M 32 58.5"/>
+<path style="fill-rule: evenodd; stroke: none; fill-opacity: 1" d="M 95 52.1 C 95.8 52.1 96.6 52.2 97.3 52.1 C 97.8 52.1 98 52.3 98 52.8 C 97.9 54.2 98 55.6 97.9 57 C 97.9 57.7 98.1 57.9 98.8 57.9 C 100.1 57.8 101.5 57.9 102.9 57.8 C 103.5 57.8 103.7 57.9 103.7 58.6 C 103.7 60.1 103.7 61.6 103.7 63.2 C 103.7 63.7 103.5 63.9 103 63.9 C 101.6 63.8 100.2 63.9 98.7 63.8 C 98.1 63.8 97.9 64 97.9 64.6 C 98 66 97.9 67.4 98 68.8 C 98 69.4 97.8 69.7 97.2 69.7 C 95.7 69.6 94.2 69.6 92.7 69.7 C 92.1 69.7 91.9 69.5 91.9 68.9 C 92 67.4 91.9 66 92 64.5 C 92 64 91.8 63.8 91.3 63.8 C 89.9 63.9 88.5 63.8 87.1 63.9 C 86.3 63.9 86.2 63.6 86.2 63 C 86.2 61.6 86.3 60.2 86.2 58.8 C 86.1 58 86.5 57.8 87.2 57.8 C 88.6 57.9 89.9 57.8 91.3 57.9 C 91.8 57.9 92 57.7 92 57.2 C 91.9 55.8 92 54.3 91.9 52.9 C 91.9 52.3 92.1 52.1 92.7 52.1 C 93.5 52.2 94.2 52.1 95 52.1 Z M 95 52.1"/>
+</g>
+</svg>
```

Comment:
The SVG can be optimized using the [SVG Viewer](https://www.svgviewer.dev/).
By the way: this icon is missing the `TM` on the corner.
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 128 128" width="128" height="128"><path style="fill-rule:evenodd;stroke:none;fill-opacity:1" d="M.3 66.5v-9.6c.2-.1.2-.3.2-.5 3-13.1 11.2-21 24.2-24.1 1.1-.3 2.3-.2 3.4-.6h6.4c.1.2.4.1.6.1 6.4.7 12.2 3 17.3 7 4.4 3.3 7.8 7.5 10.9 12 .4.6.6.6 1 0 1.8-2.6 3.7-5.1 5.9-7.4 5.3-5.7 11.7-9.7 19.5-11.1 1.1-.3 2.4-.2 3.5-.6h6.2c.1.2.3.1.5.1 1.9.2 3.7.6 5.5 1.1 13.4 3.9 22.9 16.2 22.1 30.1-.6 11.7-6.5 20.1-16.8 25.4-5.1 2.8-10.7 3.5-16.5 3.4-7.6-.1-14.2-2.7-19.9-7.7-4-3.5-7.1-7.7-10-12.1-.4-.6-.6-.5-1 .1-1.8 2.7-3.7 5.4-5.9 8-3.9 4.4-8.4 8-14 9.9-6.9 2.4-13.9 2.5-20.9.6-10.1-2.9-17-9.3-20.8-19.1-.6-1.6-.9-3.4-1.4-5zm31.8 14.7c5.7.2 10.6-1.7 14.8-5.6 4.3-4 7.4-9 10.5-13.9.1-.3.1-.5-.1-.8-2.6-4.1-5.3-8.2-8.9-11.6-6.9-6.6-15-8.8-24.1-5.9-7.5 2.5-12.3 7.8-13.4 15.8-1.1 7.5 1.8 13.5 7.8 18 4 2.9 8.5 4.1 13.4 4zm63.4 0c2.2 0 4.4-.1 6.5-.7 7.9-2.4 13.1-7.3 14.6-15.5 1.5-8.1-1.6-14.6-8.4-19.2-7.5-5.2-18.4-4.7-26 1-5.1 3.8-8.5 8.9-11.9 14.2-.2.3-.1.5 0 .8 2.7 4.3 5.4 8.6 8.9 12.3 4.4 4.7 9.7 7.4 16.3 7.1zm0 0"/><path style="fill-rule:evenodd;stroke:none;fill-opacity:1" d="M32 58.5c3.2 0 6.5.1 9.7 0 .8 0 .9.2.9 1-.1.9-.1 1.8 0 2.7.1.8-.1 1-1 1H28.8c-2.2 0-4.4-.1-6.6 0-.7 0-.9-.2-.9-1 .1-.9.1-1.8 0-2.8 0-.7.2-.9.9-.9 3.2.1 6.5 0 9.8 0zm63-6.4c.8 0 1.6.1 2.3 0 .5 0 .7.2.7.7-.1 1.4 0 2.8-.1 4.2 0 .7.2.9.9.9 1.3-.1 2.7 0 4.1-.1.6 0 .8.1.8.8v4.6c0 .5-.2.7-.7.7-1.4-.1-2.8 0-4.3-.1-.6 0-.8.2-.8.8.1 1.4 0 2.8.1 4.2 0 .6-.2.9-.8.9-1.5-.1-3-.1-4.5 0-.6 0-.8-.2-.8-.8.1-1.5 0-2.9.1-4.4 0-.5-.2-.7-.7-.7-1.4.1-2.8 0-4.2.1-.8 0-.9-.3-.9-.9 0-1.4.1-2.8 0-4.2-.1-.8.3-1 1-1 1.4.1 2.7 0 4.1.1.5 0 .7-.2.7-.7-.1-1.4 0-2.9-.1-4.3 0-.6.2-.8.8-.8.8.1 1.5 0 2.3 0zm0 0"/></svg>

```